tool/gravity/cli/operation.go

```diff
@@ -179,7 +179,7 @@ func completeOperationPlanForOperation(localEnv *localenv.LocalEnvironment, envi
 	case ops.OperationUpdateConfig:
 		err = completeConfigPlanForOperation(localEnv, environ, op)
 	default:
-		return trace.BadParameter("operation type %q does not support plan completion", op.Type)
+		return completeClusterOperationPlan(localEnv, op)
```

Comment:
What's gonna happen when this is called for "shrink"? And why the rest of "plan-based" operations are not a part of the above switch?

Reply:
For shrink, it will set the operation state to `failed` and transition the cluster state accordingly.
I'll add the handling for garbage collection operation though.